ceph/ceph_admin/apply.py

```diff
@@ -30,3 +41,51 @@ def apply(self, role, command, placements):
                 daemon=role,
                 ids=placements,
             )
+
+    def orch_remove(self, **config):
+        """
+        Remove service using "ceph orch rm <svc-id>" command
+        Args:
+            config: service remove arguments
+
+        config:
+            service_name: service name
+            check_status: check command status
+        """
+        service_name = config.get("service_name")
```

Comment:
Instead of checking, we could do `service_name = config['service_name']`... this would raise an error.

Reply:
I think it would not require, if we use,
```
def orch_remove(svc_name, check_status=True):
```